### DISCUSSION TITLE
I can't find the Kbuild file in the Virtual Machine

### DISCUSSION THREAD

**Discussion by tjt22** [bitcraze/discussions#281] [Q&A] [answered]
I'm following this tutorial: https://www.bitcraze.io/documentation/repository/crazyflie-firmware/master/development/howto/ on the Bitcraze VM and I can't seem to find the Kbuild file in src/deck/drivers/src/ to add obj-y += hello.o to. However, in the github, it is there. Not sure if I'm doing something wrong.

**Comment by krichardsson** [comment, marked as answer]
It sounds like you maybe not have the latest version of the source code? In the "[Update source code](https://www.bitcraze.io/documentation/tutorials/getting-started-with-development/#update)" step in the "Getting started with development" shows you how to do it.

**Reply by tjt22** [reply]
That worked, thanks so much!
